languages/csharp/exercises/concept/numbers-floating-point/.docs/introduction.md

```diff
@@ -10,4 +10,11 @@ C# has three floating-point types:
 
 As can be seen, each type can store a different number of digits. This means that trying to store PI in a `float` will only store the first 6 to 9 digits (with the last digit being rounded).
 
-In this exercise you may also want to use a loop. There are several ways to write loops in C#, but the `while` loop is most appropriate here.
+In this exercise you may also want to use a loop. There are several ways to write loops in C#, but the `while` loop is most appropriate here:
+
+```csharp
+while (x > 10)
```

Comment:
Should syntax examples be minimally working? i.e. in this case, should there be included a new first line of `int x = 0` ?

Reply:
Good idea. I've added a variable. 

The syntax example should not necessarily be minimally working.